master/custom/workers.py

```diff
@@ -57,7 +57,7 @@ def get_workers(settings):
         cpw(
             name="aixtools-aix-power6",
             tags=['aix', 'unix', 'power6'],
-            branches=['3.8', '3.9', '3.x'],
+            branches=['3.8', '3.9', '3.10', '3.x'],
```

Comment:
Shouldn't 3.10 be added globally just above https://github.com/python/buildmaster-config/blob/d7f396dea5fbcb08c486fc492534bc29ad4fb0e9/master/master.cfg#L121 ?